### ISSUE TITLE
Production-ready versioning + docker + migrations

### ISSUE THREAD

**Issue by omid** [fmeringdal/nettu-scheduler#16]
Seems like:

- there is no git versioning (tagging)
- the existing docker is only for development
- there is no migration, only the `dbinit.sql`

We need to have a production-ready code here, with proper versioning, production-ready docker, and migration scripts.

**Comment by fmeringdal** [comment]
Yep, definitely some work needed to make it more production ready. I think the server should follow SemVer versioning, the same goes with the SDK. 
Do you think we need a separate binary to handle the migration process?

**Comment by omid** [comment]
I would say yes.

Because normally we need to run it while deployment, and in case we have several instances of the server, it will be executed several times.

**Comment by omid** [comment]
I address the migration part of this issue, within #23
